rpm/selinux_springboot.spec

```diff
@@ -0,0 +1,52 @@
+Name:		springboot-selinux
+Version:	#{version}#
+Release:        1%{?dist}
```

Comment:
How do you want to name it?

Reply:
Not sure yet if I want the distro in the RPM release name...